src/borg/item.pyx

```diff
@@ -325,6 +325,21 @@ class Key(PropDict):
     tam_required = PropDict._make_property('tam_required', bool)
 
 
+def mft_encode(t):
+    """encode a manifest tuple"""
+    return t
+
+
+def mft_decode(t):
+    """decode a manifest tuple"""
+    # we get bytes objects from msgpack.unpack, but want str
```

Comment:
msgpack legacy not roundtripping py3 str correctly requires this.

Reply:
without this, json mode output encode falls over the bytes object returned by msgpack.